src/main/java/core/basesyntax/operation/BalanceOperation.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.operation;
+
+import java.util.Map;
+
+public class BalanceOperation implements OperationHandler {
+    @Override
+    public void execute(Map.Entry<String, Integer> entry, int value) {
+        entry.setValue(value);
+    }
+
+    public static class ReturnOperation implements OperationHandler {
```

Comment:
The inner class `ReturnOperation` should be static only if it doesn't require access to an instance of the enclosing class. If that's not the case, consider removing the `static` modifier.